pages/data-design/groups.mdx

```diff
@@ -0,0 +1,132 @@
+This is supported by Avo, Amplitude, Mixpanel, and additionally you can use this method for your data warehouse via custom destinations in Avo. This is not supported by Segment.
+
+**Consequences**:
+
+For Mixpanel and Amplitude: All events logged with a group will be available for group analyses such as calculating metrics based on unique groups. See docs for Mixpanel and Amplitude.
+
+For your data warehouse: The events will have the Group ID associated with them. If you maintain a table with each group’s metadata (see Update Group Properties section), you can additionally enrich your events with the current state of the group metadata. This will allow you to query your events by the historical state of the group metadata, as it was at the time of the event.
+
+This is not supported by Segment.
+
+**Use case**: You should log events with a group when you have product concepts where users can be associated with many groups through their lifetime.
+
+_Example_: If you have "Branches" in your product, of which users can create many through their lifetime, then you can create a "Branches" group to understand the lifetime of a branch, for example time from branch open to branch closed, etc. Then, instead of permanently associating the user profile with a specific branch, you associate only the branch related events to specific branch IDs.
+
+_Example_: If you have chat channels, you can log the “Message Sent” event with the “Channel” group and the specific channel ID, so you can do analysis of channels such as retention by channel (as opposed to retention by individual users).
+
+**How to manage in Avo**: Add the "Log Event" action type to an Avo event. Then add the Group you want to associate this event with.
+
+### Update Group Properties
+
+Conceptually the group properties are quite similar to user properties. You can maintain group metadata, aka group properties, for the group IDs that you associate users or events with.
+The first step is defining your groups (group types and group ids). Then you can attach and update specific properties about the groups.
+
+**Functionality**: This action will update metadata about a specific group, by updating the group metadata (e.g. member count of a team). This is supported by Avo, Segment, Mixpanel and Amplitude, and additionally you can use this method for your data warehouse via custom destinations in Avo.
+
+**Consequences**:
+
+[//]: # 'TODO [TO CONFIRM] For Mixpanel and Amplitude: Future events associated with this specific group will have the updated metadata about the group available for filtering, segmentation, etc.'
+
+For Mixpanel and Amplitude: Specific group will have the updated metadata about the group available for filtering, segmentation, etc.
+
+For Segment: You can only update group properties on the user level, not on the event level. This means that when you update group properties, it will automatically associate the user permanently with this group.
```

Comment:
```suggestion
For Segment: Note that you can not update group properties without associating the current user with that group. This means that when you update group properties, it will automatically associate the user permanently with this group.
```